mocks/mocks.py

```diff
@@ -62,6 +62,42 @@ def crop_simulation_data(particle_pos,
     return pos, mass
 
 
+def init_mocks(config):
+    """
+    Creates some initials for the code. This function will be converted to
+    the __init__ when the code uses class.
+
+    Parameters
+    ----------
+    config : dict
+        A dictionary of configuration items.
+
+    Returns
+    -------
+    config : dict
+        A dictionary of configuration items.
+    huntsman : gunagala.imager.Imager
```

Comment:
don't use huntsman here -- keep it generic so the docstring is relevant to any telescope we use

Reply:
I changed it to `telescope`.